dev/image-guidelines.md

```diff
@@ -0,0 +1,329 @@
+# Guidelines for creating image content
```

Comment:
We need to add a hugo preamble here to render

Reply:
This doc is not intended to be published via Hugo, at least not any time soon. It's just developer notes, similar to the ones in the velociraptor repo that explain GUI development and translations.

At some point it might form part of a contributor guide once we figure out how to present that.

Reply:
We should publish it - maybe as a knowledge base item. 

If we dont intend people to read it then whats the point of writing it?

Reply:
The point of writing it - at this time - is so that you as I can agree on basic conventions. Basically I'm saying "This is how I do it. These are the things I think about when doing it. And can we both do it this way, or would that cause problems? Etc."

The idea of having the "dev" folder is that it's a place to share notes and supporting files, similar to the "scripts" folder but for things that aren't scripts.

The reason for making it markdown is so that it can be viewed with images and formatting: either on Github or locally. It's by coincidence that the website docs are also written in markdown.

Eventually it will be great to have a proper development guide [something like this](https://docs.excalidraw.com/docs/introduction/development) and it would be it's own section on the docs site. This could be pulled into that along with other docs from the velociraptor repo, but that is going to involve proper planning and probably a lot more explanatory writing.